/**:
  ros__parameters:
    driver:
      i_enable_ir: false
      i_enable_imu: false
      i_usb_timeout: 10000
      i_pipeline_dump: false
      i_calibration_dump: false
      i_enable_diagnostics: false
      i_publish_tf_from_calibration: false
    pipeline_gen:
      i_pipeline_type: RGB
      i_nn_type: "none"
      i_enable_rgbd: false
      i_enable_sync: false
    color:
      i_width: 320
      i_height: 240
      i_fps: 10.0
      i_resolution: THE_240_P
      i_interleaved: false
      i_publish_topic: true
      i_board_socket_id: 0